### PR TITLE
zoo Dockerfile added

### PR DESCRIPTION
**Pull request by Hoda1394**
zoo Dockerfile added and cli is modified accordingly but there is an issue while I want to build the image so it is not tested yet.

### CODE REVIEW

File: nobrainerzoo/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM python:3.9.13-bullseye
+
+RUN curl -sSL http://neuro.debian.net/lists/bionic.us-nh.full | tee /etc/apt/sources.list.d/neurodebian.sources.list \
```
Comment (Hoda1394) [inline nobrainerzoo/Dockerfile:3]: When I am building the docker image it gets stuck at this stage asking if the user wants to participate in the package survey? I tried to pass the answer in the command but it didn't work!
@satra & @djarecka - any idea how to solve it?

Comment (satra) [reply nobrainerzoo/Dockerfile:3]: @Hoda1394 - bullseye is a debian release and bionic is ubuntu, the two are not matched. but neurodebian site should provide the relevant command to run. if not you can also check with neurodocker.

**Comment by Hoda1394** [comment]
for adding the `cache` variable, I was thinking that we should have a priority among paths given by each variable to avoid multiple cache paths. something like this,

1. env variable `NOBRAINER_CACHE` which is set out of the zoo 
2. `cache` variable that is set from `init` function
3. `CACHE_PATH` that is set to a default value

env variable `NOBRAINER_CACHE` has a higher priority since we cannot change it from inside python and changing it is considered a security hazard. So if it is set, the `cache` variable will not be accepted. and if we have the `cache` path from `init`, the default path should be overridden. but the issue with the `cache` variable is the user should pass it each time he runs the zoo and if not, the default path will be used. any suggestion for this part?

@satra - your opinion?

**Comment by satra** [comment]
@Hoda1394 - i think this section implements this order: https://github.com/neuronets/nobrainer-zoo/blob/dev/nobrainerzoo/utils.py#L7 and init also respects this by default: https://github.com/neuronets/nobrainer-zoo/blob/dev/nobrainerzoo/cli.py#L471

the one thing that needs to be added to init is that it should set an environment variable here: https://github.com/neuronets/nobrainer-zoo/blob/dev/nobrainerzoo/cli.py#L481

if the cache is needed inside the container, it should be passed on via an environment variable and the location of the cache should also be bound appropriately to the container.

**Pull request merged**